models/order.js

```diff
@@ -12,32 +12,41 @@ const orderSchema = new Schema({
 });
 
 orderSchema.statics.getOrders = function() {
-  return getOrders.apply(this)
-  .then(getEntries)
-  .catch((e) => {
-    throw new Error(e);
```

Comment:
I think you can catch in transactAndDispatch


> On Nov 24, 2015, at 10:50, Andy Chiu <notifications@github.com> wrote:
> 
> In models/order.js:
> 
> > @@ -12,32 +12,41 @@ const orderSchema = new Schema({
> >  });
> >  
> >  orderSchema.statics.getOrders = function() {
> > -  return getOrders.apply(this)
> > -  .then(getEntries)
> > -  .catch((e) => {
> > -    throw new Error(e);
> Whoops.. I'll add back. I should be able to catch in transactAndDispatch right because errors get passed on? Or do you think I should catch at each step?
> 
> —
> Reply to this email directly or view it on GitHub.
>